airflow/providers/amazon/aws/example_dags/example_local_to_s3.py

```diff
@@ -0,0 +1,42 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+
+import os
+
+from airflow import models
+from airflow.providers.amazon.aws.transfers.local_to_s3 import LocalFilesystemToS3Operator
+from airflow.utils.dates import datetime
+
+S3_BUCKET = os.environ.get("S3_BUCKET", "test-bucket")
```

Comment:
Nitpick for discussion, non-blocking:  There are currently four example DAGS (not counting this one) related to S3.  Two of them use the envvar "S3_BUCKET" and the other two use "BUCKET_NAME".  Should we standardize those and, if so, which way?

Reply:
I would opt for "S3_BUCKET" as the hook uses `dest_bucket`, and it seems that "S3_BUCKET" over "BUCKET_NAME" more closely follows that precedent.

Reply:
Let's do this as a separate contribution because it makes sense to standardize all the variables and write it down as a generally accepted rule to make it more permanent. Now, if we change the variable name in one place, we will fix the problem only here, but in a moment subsequent contributions may not see this convention and introduce a different naming convention.